src/domain/SuggestionPage/fields/TranslatedFieldInput.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { FormikHandlers } from 'formik';
+import { TextArea, TextInput } from 'hds-react';
+
+import styles from '../SuggestionForm.module.scss';
+import { Tab, Tabs } from '../../../common/ui-components/Tabs/Tabs';
+import { TranslatedField } from '../types';
+
+interface TranslatedFieldInputProps {
+  readonly labelText: string;
+  readonly id: string;
+  readonly value: TranslatedField;
+  readonly handleChange: FormikHandlers['handleChange'];
+  readonly textArea?: boolean;
+  readonly error?: string;
+}
+
+const TranslatedFieldInput: React.FC<TranslatedFieldInputProps> = ({
+  labelText,
+  id,
+  value,
+  handleChange,
+  textArea,
+  error,
+}) => {
+  const Input = textArea ? TextArea : TextInput;
+  return (
+    <div>
```

Comment:
```suggestion
    <>
```

Reply:
Changed to include a custom `className` instead to improve reuse.